etcdctl/README.md

```diff
@@ -26,7 +25,7 @@ PUT assigns the specified value with the specified key. If key already holds a v
 
 RPC: Put
 
-#### Options
+#### PUT options \key\ \value\ Options
```

Comment:
I suggest that we ignore the linter rule instead of updating this.

Reply:
Agreed, the lint fixes were creating too much noise. I reverted them and instead used an inline markdownlint-disable comment to suppress the existing errors. The PR should be much cleaner now.